core/src/avm2/domain.rs

```diff
@@ -182,35 +182,27 @@ impl<'gc> Domain<'gc> {
 
     /// Export a definition from a script into the current application domain.
     ///
-    /// This returns an error if the name is already defined in the current or
-    /// any parent domains.
+    /// This does nothing if the definition already exists.
     pub fn export_definition(
         &mut self,
         name: QName<'gc>,
         script: Script<'gc>,
         mc: MutationContext<'gc, '_>,
-    ) -> Result<(), Error<'gc>> {
+    ) {
         if self.has_definition(name) {
-            return Err(format!(
-                "VerifyError: Attempted to redefine existing name {}",
-                name.local_name()
-            )
-            .into());
+            return;
```

Comment:
Can you point out how this situation is handled in avmplus? I wonder whether the first defined trait remains or later traits override each other.

Reply:
I've added a test that proves it ignores the second definition